common/faults/fault_config.json

```diff
@@ -64,40 +64,94 @@
                     "lcd_message": "HIGH PDU MCU TEMP"
                 },
                 {
-                    "fault_name": "lv_bat_low",
-                    "max": 34,
-                    "min": 24,
+                    "fault_name": "dt_l_temp_ot",
+                    "max": 90,
+                    "min":-100,
                     "priority": "warning",
                     "time_to_latch": 1000,
-                    "time_to_unlatch": 2000,
-                    "lcd_message": "LV Bat Getting Low"
+                    "time_to_unlatch": 1000,
+                    "lcd_message": "Left Drivetrain Overheating"
                 },
                 {
-                    "fault_name": "lv_bat_very_low",
-                    "max": 34,
-                    "min": 22,
-                    "priority": "error",
+                    "fault_name": "dt_r_temp_ot",
+                    "max": 90,
+                    "min":-100,
+                    "priority": "warning",
                     "time_to_latch": 1000,
-                    "time_to_unlatch": 2000,
-                    "lcd_message": "LV Bat Very Low"
+                    "time_to_unlatch": 1000,
+                    "lcd_message": "Right Drivetrain Overheating"
                 },
                 {
-                    "fault_name": "lv_bat_bms",
-                    "max": 1,
-                    "min": 0,
-                    "priority": "error",
-                    "time_to_latch": 50,
-                    "time_to_unlatch": 2000,
-                    "lcd_message": "LV Bat BMS Fault"
+                    "fault_name": "dt_l_temp_high",
+                    "max": 80,
+                    "min":-100,
+                    "priority": "warning",
+                    "time_to_latch": 1000,
+                    "time_to_unlatch": 1000,
+                    "lcd_message": "Left Drivetrain High (>80C)"
                 },
                 {
-                    "fault_name": "drive_flow",
-                    "max": 63,
-                    "min": 46,
-                    "priority": "error",
-                    "time_to_latch": 10,
-                    "time_to_unlatch": 10,
-                    "lcd_message": "battery pump flow brokey"
+                    "fault_name": "dt_r_temp_high",
+                    "max": 80,
+                    "min":-100,
+                    "priority": "warning",
+                    "time_to_latch": 1000,
+                    "time_to_unlatch": 1000,
+                    "lcd_message": "Right Drivetrain High (>80C)"
```

Comment:
Are these off of the DT gearbox thermistor or the temp from the motor controller? Maybe specify in the message.